home/src/main/resources/rdf/dynapiAbox/everytime/paramerter_types.n3

```diff
@@ -0,0 +1,141 @@
+
+<https://vivoweb.org/ontology/vitro-dynamic-api/parameter/type/string-literal>
+        a                       dynapi:ParameterType ;
+        dynapi:name             "string" ;
+        rdfs:label              "String literal type"@en-US ;
+        dynapi:serializationType <https://vivoweb.org/ontology/vitro-dynamic-api/parameter/serializedtype/string> ;
+        dynapi:rdfType		<https://vivoweb.org/ontology/vitro-dynamic-api/parameter/rdftype/string> ;
+        dynapi:implementationType <https://vivoweb.org/ontology/vitro-dynamic-api/parameter/implementation-type/string-literal> ;
+        vitro:mostSpecificType  dynapi:ParameterType .
+
+<https://vivoweb.org/ontology/vitro-dynamic-api/parameter/type/model>
+        a                       dynapi:ParameterType ;
+        dynapi:name             "model" ;
+        rdfs:label              "Model type"@en-US ;
+        dynapi:implementationType <https://vivoweb.org/ontology/vitro-dynamic-api/parameter/implementation-type/model> ;
+        dynapi:serializationType <https://vivoweb.org/ontology/vitro-dynamic-api/parameter/serializedtype/string> ;
+        vitro:mostSpecificType  dynapi:ParameterType .
+
+<https://vivoweb.org/ontology/vitro-dynamic-api/parameter/type/internal-model>
+        a                       dynapi:ParameterType ;
+        dynapi:name             "internal model" ;
+        rdfs:label              "Model type"@en-US ;
+        dynapi:isInternal		"true"^^xsd:boolean ;
+        dynapi:implementationType <https://vivoweb.org/ontology/vitro-dynamic-api/parameter/implementation-type/internal-model> ;
+        dynapi:serializationType <https://vivoweb.org/ontology/vitro-dynamic-api/parameter/serializedtype/string> ;
+        vitro:mostSpecificType  dynapi:ParameterType .
+
+<https://vivoweb.org/ontology/vitro-dynamic-api/parameter/type/JsonContainer>
+        a                       dynapi:ParameterType ;
+        dynapi:name             "json container" ;
+        rdfs:label              "Object type"@en-US ;
+        dynapi:implementationType <https://vivoweb.org/ontology/vitro-dynamic-api/parameter/implementation-type/json-container> ;
+        dynapi:serializationType <https://vivoweb.org/ontology/vitro-dynamic-api/parameter/serializedtype/json-container> ;
+        vitro:mostSpecificType  dynapi:ParameterType .
+
+<https://vivoweb.org/ontology/vitro-dynamic-api/parameter/type/JsonContainerArray>
+        a                       dynapi:ParameterType ;
+        dynapi:name             "json array" ;
+        rdfs:label              "Array type"@en-US ;
+        dynapi:implementationType <https://vivoweb.org/ontology/vitro-dynamic-api/parameter/implementation-type/json-container-array> ;
+        dynapi:serializationType <https://vivoweb.org/ontology/vitro-dynamic-api/parameter/serializedtype/json-container> ;
+        vitro:mostSpecificType  dynapi:ParameterType .
+
+<https://vivoweb.org/ontology/vitro-dynamic-api/parameter/type/JsonContainerArrayTest>
```

Comment:
Is this needed?

Reply:
It is used by dynamic-api-individuals-testing.n3